sql/moz-fx-data-shared-prod/monitoring_derived/structured_missing_columns_v1/query.py

```diff
@@ -58,16 +58,30 @@
     document_type,
     document_version,
     path
+),
+formatted_path AS (
```

Comment:
Looks like this CTE could be avoided if we just add this udf call inside `transformed`?